packages/amplify_auth_cognito/lib/method_channel_auth_cognito.dart

```diff
@@ -320,6 +320,27 @@ class AmplifyAuthCognitoMethodChannel extends AmplifyAuthCognito {
     }
   }
 
+  @override
+  Future<Map<String, UpdateUserAttributeResult>> updateUserAttributes(
+      {required UpdateUserAttributesRequest request}) async {
+    Map<String, UpdateUserAttributeResult> res;
+    try {
+      final Map<String, dynamic>? data =
+          await _channel.invokeMapMethod<String, dynamic>(
+        'updateUserAttributes',
+        <String, dynamic>{
+          'data': request.serializeAsMap(),
+        },
+      );
+      if (data == null)
+        throw AmplifyException(
+            AmplifyExceptionMessages.nullReturnedFromMethodChannel);
```

Comment:
Nit: Wrap if with curly braces for readability
```suggestion
      if (data == null) {
        throw AmplifyException(
            AmplifyExceptionMessages.nullReturnedFromMethodChannel);
      }
```

Reply:
Thanks added {}